users/models.py

```diff
@@ -93,3 +93,24 @@ class Apartment(models.Model):
 
     def __str__(self):
         return f"Owner:{self.owner}, Addres:{self.address}, City:{self.city}"
+
+
+class Preferences(models.Model):
+    user = models.OneToOneField(
```

Comment:
Why use a OneToOne Relationship instead of simply adding the fields into the User model? 
There are generally 2 reasons to use a OneToOne:
1) Not all the records in the primary table habe records in the secondary table.
2) The primary table comes from an external source and you can't change it..

In this case, I believe that all your users will have Preferences and you wrote the user model. 
Adding the fields directly to the user model will also solve your problem as to whether to call them preferences or not, because they will just be additional fields without any special name.

Reply:
Thank you for the explanation.
This is done.

Reply:
Can you approve the changes?